src/cpp/benders/benders_core/BendersMathLogger.cpp

```diff
@@ -0,0 +1,77 @@
+#include "BendersMathLogger.h"
+
+#include "LogUtils.h"
+#include "LoggerUtils.h"
+
+HeadersManager::HeadersManager(HEADERSTYPE type, const BENDERSMETHOD& method) {
+  headers_list.push_back("ITE");
+  headers_list.push_back("LB");
+  if (method == BENDERSMETHOD::BENDERS) {
+    headers_list.push_back("UB");
+    headers_list.push_back("BESTUB");
+    headers_list.push_back("AGAP");
+    headers_list.push_back("RGAP");
+  }
```

Comment:
To be more uniform and explicit in the printings, I suggest :
- `Ite`
- `Lb`
- `Ub`
- `BestUb`
- `AbsGap`
- `RelGap`

Reply:
done